homeassistant/components/intellifire/entity.py

```diff
@@ -0,0 +1,39 @@
+from __future__ import annotations
+
+from datetime import datetime
+
+from intellifire4py import IntellifirePollData
+
+from homeassistant.components.intellifire import IntellifireDataUpdateCoordinator
+from homeassistant.helpers.entity import EntityDescription
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+from collections.abc import Callable
+from dataclasses import dataclass
+
+
+@dataclass
+class IntellifireEntityDescription(EntityDescription):
```

Comment:
I'm not sure this class is actually useful because it doesn't really do much (right now)